explorer/next.config.js

```diff
@@ -1,6 +1,10 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  images: {
```

Comment:
I'm not familiar with imgix! Can you tell me why you have added it (and what is the noop path)?

Reply:
This might not even be used. At the moment I'm just using cuz when we use next export the default image loader doesn't work and you need to specify a new one, otherwise, you'll get an error when trying to build the project.
This might change depending on how we will handle images' size.